src/components/Confirm/index.js

```diff
@@ -94,14 +90,20 @@ const Actions = ({retakeAction, confirmAction, error}) =>
     </div>
   </div>
 
-const Previews = ({capture, retakeAction, confirmAction, error}) =>
-  <div>
-    {error.type ? <Error error={error} /> : <PreviewHeader />}
-    <div className={theme.imageWrapper}>
-      <CaptureViewer capture={capture} />
-      <Actions retakeAction={retakeAction} confirmAction={confirmAction} error={error} />
+const Previews = ({capture, retakeAction, confirmAction, error, method, documentType}) => {
+  const title = confirm[method].title
+  const subTitle = method === 'document' ? confirm[documentType] : confirm.face
```

Comment:
where was this `method` logic done before? keeping business logic at a higher level helps keep lower level components dumber and easier to follow

Reply:
This logic did not exist before. Before we used the same copy on all confirmation screens, this PR introduces specific copy for each document type and selfie.

This logic could be moved up to the `Confirm` component, but in the case of an error we don't need to generate the copy. Therefore I put it in the Preview component so it is only evaluated when we need it. Do you think it should go in `Confirm`?

Reply:
if it didn't exist before, it's fine I think. It has to go somewhere.